wire-tests-swift/src/main/swift/module_two/SwiftModuleTwoMessage.swift

```diff
@@ -0,0 +1,104 @@
+// Code generated by Wire protocol buffer compiler, do not edit.
+// Source: squareup.protos.kotlin.swift_module_two.SwiftModuleTwoMessage in swift_module_two.proto
+import Foundation
+import Wire
+import module_one
+
+public struct SwiftModuleTwoMessage {
+
+    public var name: String?
+    public var module_one_names: [SwiftModuleOneMessage] = []
```

Comment:
This needs to be a Heap type, so you need ~16 parameters in the type

Reply:
The error reproduces without it I believe, still working on it.